### ISSUE TITLE
No output

### ISSUE THREAD

**Issue by platinum55** [rtravis/SkypeCacheViewer#3]
Hi,

Thanks for this project! I successfully compiled the code and executed as mentioned, but there is no output at all. What kind of output is expected? 
Tried to look with strace whats happening, but I could not find any output file written

Thanks for advice :)

**Comment by rtravis** [comment]
Did you provide the executable the path to a valid Skype LevelDB path? It should be a the path to a directory that ends with: ".indexeddb.leveldb". By default the program should display all Skype contacts that are found in the LevelDB argument.

**Comment by platinum55** [comment]
yes, did that. In those files I see some kind of json format, so there is data - maybe new format? thanks

**Comment by rtravis** [comment]
I was not able to replicate this issue.

I know that the databases might contain confidential data, however, if you have a database that is safe to share can you please make an archive and provide it as a web download?
